model_cards/deepset/gbert-large/README.md

```diff
@@ -0,0 +1,54 @@
+---
+language: de
+license: mit
+datasets:
+- wikipedia
+- OPUS
+- OpenLegalData
+- OSCAR
```

Comment:
datasets ids should be lowercase, not sure if we post-process those ids in model cards